File: C#/Raviraj Padhiyar/DAY5/Assignment/Mobike.cs
using System;

namespace DemoConsoleApp
{
    class Mobike
    {
        public int BikeNumber { get; set; }
        public string PhoneNumber { get; set; }
        public string Name { get; set; }
        public int Days { get; set; }
        public decimal Charge { get; set; }


        public void Input(int bikeNumber, string phoneNumber, string name,int days)
        {
            BikeNumber = bikeNumber;
            PhoneNumber = phoneNumber;
            Name = name;
            Days = days;
            Compute();
        }

        public void Compute()
        {
            int cost = 0;
            for (int i = 0; i < this.Days; i++)
            {
                if (i < 5)
                {
                    cost += 500;
                }
                else if (i < 10)
                {
                    cost += 400;
                }
                else
                {
                    cost += 200;
                }
            }

            Charge = cost;
        }

        public void Display()
        {
            Console.Write($"{BikeNumber}\t{PhoneNumber}\t{Days}\t{Charge}");
        }
    }
}

File: NodeJs/Hardik Niranjani/Day3/src/Assignment/Mobike.js
class Mobike{
    constructor(BikeNumber,PhoneNumber,Name,Days){
        this.BikeNumber = BikeNumber;
        this.PhoneNumber = PhoneNumber;
        this.Name = Name;
        this.Days = Days;
    }

    compute(){
        var res = 0;
        var Day5 = 2500;
        var Day10 = 2000;
        if(this.Days <=5){
            res = 500*(this.Days);
        }
        else if (this.Days >5 && this.Days<=10){
            res = Day5 + 400*(this.Days - 5);
        }

        else {
            res = Day5 + Day10 + 200*(this.Days - 10);
        }
        return res;
     }
}

module.exports = Mobike;